OpenSim/Examples/DataTable/example.cpp

```diff
@@ -0,0 +1,61 @@
+/* -------------------------------------------------------------------------- *
+ *                            OpenSim:  example.cpp                           *
```

Comment:
How about renaming `dataTableExamples.cpp`? We should add many more examples and usages here.